packages/core/src/Site/index.js

```diff
@@ -1282,15 +1275,18 @@ class Site {
   }
 
   /**
-   * Copies bootswatch theme to the assets folder if a valid theme is specified
+   * Copies bootstrap theme to the assets folder if a valid theme is specified
+   * @param {Boolean} toCopyDefault bootstrap theme to the assets folder
    */
-  copyBootswatchTheme() {
+  copyBootstrapTheme(toCopyDefault) {
     const { theme } = this.siteConfig;
-    if (!theme || !_.has(SUPPORTED_THEMES_PATHS, theme)) {
+    if (!toCopyDefault && (!theme || !_.has(SUPPORTED_THEMES_PATHS, theme))) {
```

Comment:
Minor nit: would it be possible to remove the need for the `toCopyDefault`? It seems like if `toCopyDefault` is True and `theme` is defined, then it would copy the theme's file instead of the default file. 

What do you think of the following flow:
If `theme` is undefined, then we will assume that the caller wants to copy the default bootstrap theme.
Else check if `theme` is a supported theme and copy over the relevant theme file.

Reply:
> What do you think of the following flow:
> If theme is undefined, then we will assume that the caller wants to copy the default bootstrap theme.
> Else check if theme is a supported theme and copy over the relevant theme file.

Hmm I think this will make a redundant copy of the default theme if no theme is specified in the initial build. Maybe I can rename `toCopyDefault` to `isRebuild` so that it is more concise. What do you think?

Reply:
> Hmm I think this will make a redundant copy of the default theme if no theme is specified in the initial build. 

Hmm, I see. In that case, should we log a warning / throw exception for the case `toCopyDefault = True && theme`?

> Maybe I can rename toCopyDefault to isRebuild so that it is more concise. What do you think?

I am fine with either name.

Reply:
Sorry I think I wasn't too clear about this. I think I should rename `toCopyDefault` as it is a little misleading. This parameter should only be True if it is a rebuild (site.json was modified).

What do you think about this check:
```
if ((!isRebuild && !theme) || (theme && !_.has(SUPPORTED_THEMES_PATHS, theme))) {
      return _.noop;
}
```
It will be a `noop` during the first build if theme is not specified/theme is invalid. Else, if it is subsequent rebuild/theme is specified and valid, then, it will copy over the relevant theme's file (including default theme if `!theme`).

Reply:
Yes that looks good! Can you also add a comment next to this check to indicate its purpose for future reference?

Reply:
Alright I've updated this and also added the comment :)